crates/solidity/inputs/language/src/definition.rs

```diff
@@ -495,7 +495,7 @@ codegen_language_macros::compile!(Language(
                             )]
                         ),
                         Keyword(
-                            name = BytesKeyword,
+                            name = FixedBytesType,
```

Comment:
I think the official grammar itself is not consistent about this, for example `BoolKeyword` or `StringKeyword`.
Since they are syntactically keywords (reserve their values from creating identifiers), I think we should should treat them as such?